OverlayPlugin.Core/resources/reserved_log_lines.json

```diff
@@ -0,0 +1,28 @@
+[
+    {
+        "StartID": 0,
+        "EndID": 255,
+        "Source": "FFXIV_ACT_Plugin",
+        "Version": 0,
+        "Comment": "Reserved for base FFXIV_ACT_Plugin use"
+    },
+    {
+        "ID": 256,
+        "Source": "OverlayPlugin",
+        "Version": 1,
+        "Comment": "Line to be emitted when a new log line is registered"
+    },
+    {
+        "ID": 257,
+        "Source": "OverlayPlugin",
+        "Version": 1,
+        "Comment": "MapEffect data"
+    },
+    {
+        "StartID": 258,
+        "EndID": 512,
```

Comment:
I still am a little unsure if this is needed? I think if people send PRs to register lines here, that should be plenty of coordination without needing to sort of predict or preallocate space here.

Reply:
My main thought here is from an at-a-glance/organization standpoint I don't want to see lines intertwined all over the place. But that discussion belongs on the other PR in general, I think.